wire-ios/Wire-iOS/Sources/UserInterface/GroupDetails/GroupDetailsViewController.swift

```diff
@@ -122,7 +122,7 @@ final class GroupDetailsViewController: UIViewController, ZMConversationObserver
         navigationItem.titleView = TwoLineTitleView(
             first: L10n.Localizable.Participants.title.capitalized.attributedString,
             second: verificationStatus)
-        navigationItem.rightBarButtonItem = UIBarButtonItem.closeButton(action: { [weak self] _ in
+        navigationItem.rightBarButtonItem = UIBarButtonItem.closeButton(action: UIAction { [weak self] _ in
```

Comment:
Not a request for change but an observation.... 

As this is repeated a lot I think it is worth generalizing it. Something like:

```swift
static func closeButton(viewController: UIViewController) -> UIBarButtonItem
```

Reply:
We discussed this 1:1, and we came to the conclusion that will add complexity to the method. For now we're keeping it the way it is.

Reply:
Yeah after discussing it with @agisilaos I don't think this makes sense.